canonical/ubuntu.com
# chore: desktop flavours images update on desktop/flavours

## Done

- replaced desktop flavours images on `desktop/flavours`

## QA

- Go to https://ubuntu-com-14399.demos.haus/desktop/flavours
- Check if the images from [copy doc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit) are updated correctly
- **note:** the image update on `Edubuntu` section is done in another PR

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-14870
